inventory.py

```diff
@@ -27,8 +30,13 @@ def main():
 
 
 def add_product():
+    """
+    This function adds a product into the products database.
+    The product.csv should be in the local directory relative to this program
+    """
     prod_dict = {}
 
+    #read in a CSV from storage to add existing products
     file_name = 'E:/Buds Files/Work Files/Revature/Training/Exercises/Python/Project0/product.csv'
```

Comment:
This code will currently not work on someone else's computer because you are referring to a specific file on your computer. You may want to change this.

Reply:
Yes, that's a good suggestion.  I plan on doing re-use of similar codes but I first want to complete all the functions to have the more or less complete code and then see all the common codes and refactor the code.

Reply:
Yes, this hard-coded reference to my local file should be changed to be more flexible.